package.json

```diff
@@ -42,7 +42,7 @@
     "check": "gts check",
     "clean": "gts clean",
     "compile": "tsc -p . && cp -r src/v2 build/src/v2 && cp -r protos build && cp test/*.js build/test",
-    "fix": "gts fix && eslint --fix 'samples/*.js' 'samples/**/*.js'",
+    "fix": "gts fix && eslint --fix 'samples/*.js' 'samples/**/*.js' 'src/**/*.js'",
```

Comment:
Just make this `eslint --fix '**/*.js'`.  If it picks up something it shouldn't we should have that in the `.eslintignore`.

Reply:
I am generally confused.  Why are we not formatting the generated code?

Reply:
I'm trying to make it format generated code :)

Reply:
lol sorry, misread.  Regardless, let's try to capture all the js :)